src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -215,6 +215,26 @@ bool mpuGyroRead(gyroDev_t *gyro)
     return true;
 }
 
+// Fix to make F1 fit into flash in 3.2
+#if (FLASH_SIZE > 128)
```

Comment:
This should not be required, since linker will just not include this if it is not used.